terraform/app/variables.tf

```diff
@@ -203,6 +203,12 @@ variable "add_database_name_suffix" {
   default     = false
   description = "Add optional suffix to the postgres instance name to differentiate between environments"
 }
+variable "sidekiq_replicas" {
```

Comment:
Please configure the same default values as paas: https://github.com/DFE-Digital/teaching-vacancies/blob/main/terraform/app/variables.tf#L116-L122